elements/lisk-api-client/test/unit/transaction.spec.ts

```diff
@@ -0,0 +1,716 @@
+			communityIdentifier: 'Lisk',
+			minRemainingBalance: '5000000',
+			activeDelegates: 101,
+			standbyDelegates: 2,
+			delegateListRoundOffset: 2,
+		},
+		registeredModules: [
+			{
+				id: 2,
+				name: 'token',
+				actions: [],
+				events: [],
+				reducers: [
+					'token:credit',
+					'token:debit',
+					'token:getBalance',
+					'token:getMinRemainingBalance',
+				],
+				transactionAssets: [{ id: 0, name: 'transfer' }],
+			},
+			{ id: 3, name: 'sequence', actions: [], events: [], reducers: [], transactionAssets: [] },
+			{
+				id: 4,
+				name: 'keys',
+				actions: [],
+				events: [],
+				reducers: [],
+				transactionAssets: [{ id: 0, name: 'registerMultisignatureGroup' }],
+			},
+			{
+				id: 5,
+				name: 'dpos',
+				actions: ['dpos:getAllDelegates'],
+				events: [],
+				reducers: [],
+				transactionAssets: [
+					{ id: 0, name: 'registerDelegate' },
+					{ id: 1, name: 'voteDelegate' },
+					{ id: 2, name: 'unlockToken' },
+					{ id: 3, name: 'reportDelegateMisbehavior' },
+				],
+			},
+			{
+				id: 1000,
+				name: 'legacyAccount',
+				actions: [],
+				events: [],
+				reducers: [],
+				transactionAssets: [{ id: 0, name: 'reclaimLSK' }],
+			},
+		],
+	};
+	const txHex =
+		'0802100018362080ade2042a20dd4ff255fe04dd0159a468e9e9c8872c4f4466220f7e326377a0ceb9df2fa21a321d0880ade2041214654087c2df870402ab0b1996616fd3355d61f62c1a003a4079cb29dca7bb9fce73a1e8ca28264f779074d259c341b536bae9a54c0a2e4713580fcb192f9f15f43730650d69bb1f3dcfb4cb6da7d69ca990a763ed78569700';
+	const accountHex =
+		'0a14ab0041a7d3f7b2c290b5b834d46bdc7b7eb8581512050880c2d72f1a020800220208002a3b0a1a0a0a67656e657369735f3834180020850528003080a094a58d1d121d0a14ab0041a7d3f7b2c290b5b834d46bdc7b7eb858151080a094a58d1d';
+	const encodedTx = Buffer.from(txHex, 'hex');
+	const tx = {
+		moduleID: 2,
+		assetID: 0,
+		nonce: BigInt('54'),
+		fee: BigInt('10000000'),
+		senderPublicKey: Buffer.from(
+			'dd4ff255fe04dd0159a468e9e9c8872c4f4466220f7e326377a0ceb9df2fa21a',
+			'hex',
+		),
+		asset: {
+			amount: BigInt('10000000'),
+			recipientAddress: Buffer.from('654087c2df870402ab0b1996616fd3355d61f62c', 'hex'),
+			data: '',
+		},
+		signatures: [
+			Buffer.from(
+				'79cb29dca7bb9fce73a1e8ca28264f779074d259c341b536bae9a54c0a2e4713580fcb192f9f15f43730650d69bb1f3dcfb4cb6da7d69ca990a763ed78569700',
+				'hex',
+			),
+		],
+		id: 'dd93e4ca5b48d0b604e7cf2e57ce21be43a3163f853c83d88d383032fd830bbf',
+	};
+	const passphrase1 = 'trim elegant oven term access apple obtain error grain excite lawn neck';
+	const { publicKey: publicKey1 } = getAddressAndPublicKeyFromPassphrase(passphrase1);
+	const validTransaction = {
+		moduleID: 2,
+		assetID: 0,
+		nonce: BigInt('1'),
+		fee: BigInt('10000000'),
+		senderPublicKey: publicKey1,
+		asset: {
+			recipientAddress: Buffer.from('3a971fd02b4a07fc20aad1936d3cb1d263b96e0f', 'hex'),
+			amount: BigInt('4008489300000000'),
+			data: '',
+		},
+	};
+	const txId = Buffer.from(tx.id, 'hex');
+	const schema = {
```

Comment:
can we move schema to utils folder as its making the setup part too big in this test file? maybe also nodeInfo and some other variables that are using huge space and can be reused for future tests?

Reply:
yep, what should I add them under in utils ?

Reply:
for ex, you can create `test/utils/transactions.ts` and add `export const transactionSchema = {...`, same goes for nodeInfo and other variables if they are too big and generic to be in the test